website/addons/evernote/static/evernoteLogActionList.json

```diff
@@ -0,0 +1,6 @@
+{
+  "evernote_folder_selected" : "${user} linked evernote folder ${folder} in ${node}",
```

Comment:
Please use `ADDON_FULL_NAME` casing, i.e. `Evernote`.

Also, please use addon-specific language for things that will be displayed to the user --  in this case, 'folders' should be called 'notebooks'